params.go

```diff
@@ -440,6 +446,7 @@ func parseConfig() (config.Config, error) {
 		InspectBallotActionCost:         shared.Resources(*iigoInspectBallotActionCost),
 		InspectAllocationActionCost:     shared.Resources(*iigoInspectAllocationActionCost),
 		AppointNextPresidentActionCost:  shared.Resources(*iigoAppointNextPresidentActionCost),
+		DefaultSanctionScore:            shared.IIGOSanctionsScore(*iigoDefaultSanctionScore),
```

Comment:
idk why the sequence does not match that of the vars above but i'd strive to match it so if you forgot to put it in you know immediately from the seq

Reply:
Yeah fair enough, updated in latest commit